### PR TITLE
test(web_console): E2E testing

### PR DESCRIPTION
**Pull request by marswong**

### CODE REVIEW

File: web_console/api/job.js [hunk 1]
```diff
@@ -97,18 +97,24 @@ router.get('/api/v1/job/pod/:pod_name/logs', SessionMiddleware, async (ctx) => {
...
-  const { name, job_type, client_ticket_name, server_ticket_name } = ctx.request.body;
+  const {
+    name, job_type, client_ticket_name, server_ticket_name,
+    client_params, server_params,
+  } = ctx.request.body;
 
-  const [client_params_pass, client_params] = checkParseJson(ctx.request.body.client_params);
-  if (!client_params_pass) {
+  try {
+    JSON.stringify(client_params);
```
Comment (nervouself) [inline web_console/api/job.js:106]: JSON.stringify(String) would not throw error.

File: web_console/api/job.js [hunk 2]
```diff
@@ -167,7 +173,11 @@ router.post('/api/v1/job', SessionMiddleware, async (ctx) => {
   }
   const rpcClient = new FederationClient(clientFed);
   try {
-    await rpcClient.createJob(job);
+    await rpcClient.createJob({
+      ...job,
+      client_params: JSON.stringify(client_params),
```
Comment (nervouself) [inline web_console/api/job.js:178]: client_params should not send to the other platform

File: web_console/pages/job/index.jsx [hunk 3]
```diff
@@ -130,6 +147,14 @@ function JobList(props) {
     mutate();
     toggleForm();
   };
+  const onCreateJob = (form) => {
+    const params = {
+      ...form,
+      client_params: JSON.parse(form.client_params),
+      server_params: JSON.parse(form.server_params),
```
Comment (nervouself) [inline web_console/pages/job/index.jsx:154]: The errror message would be "Unexpected token _ in JSON at position 0"

File: web_console/tests/api/federation.test.js [hunk 4]
```diff
@@ -1,52 +1,94 @@
...
+  leader = leaderRecord;
 
-    const [followerRecord] = await Federation.findOrCreate({
-      paranoid: false,
-      where: {
-        name: { [Op.eq]: follower.name },
+  const [followerRecord] = await Federation.findOrCreate({
+    paranoid: false,
+    where: {
+      name: { [Op.eq]: federations.follower.name },
```
Comment (piiswrong) [inline web_console/tests/api/federation.test.js:48]: 这个follower是啥？

**Pull request merged**